Source/WebCore/platform/audio/gstreamer/AudioFileReaderGStreamer.cpp

```diff
@@ -340,7 +345,17 @@ void AudioFileReader::decodeAudioForBusCreation()
         source = gst_element_factory_make("filesrc", nullptr);
         g_object_set(source, "location", m_filePath, nullptr);
     }
+#if PLATFORM(BROADCOM)
+    GstElement* parse = gst_element_factory_make("mpegaudioparse", NULL);
+    m_decodebin = gst_element_factory_make("brcmmp3decoder", NULL);
+
+    gst_bin_add_many(GST_BIN(m_pipeline.get()), source, parse, WTF::refGPtr(m_decodebin.get()), NULL);
 
+    gst_element_link_pads_full(source, "src", parse, "sink", GST_PAD_LINK_CHECK_NOTHING);
+    gst_element_link_pads_full(parse, "src", m_decodebin.get(), "brcm-mp3-sink", GST_PAD_LINK_CHECK_NOTHING);
+
+    plugDeinterleave(nullptr);
```

Comment:
Can we not use "pad-added" callback on the `brcmmp3decoder` element to call this method like we do on the normal decodebin?

Where's the documentation for the BROADCOM gstreamer element in use out of interest?